ij/java-psi/src/main/kotlin/org/jetbrains/academy/test/system/java/ij/analyzer/IjCodeAnalyzerUtil.kt

```diff
@@ -0,0 +1,196 @@
+package org.jetbrains.academy.test.system.java.ij.analyzer
```

Comment:
I'd suggest to separate this file into several ones, since it is loo long now, e.g.:

PsiFileAnalyzerUtil
IjCodeAnalyzerUtil
PsiElementAnalyzerUtil

What do you think?